### ISSUE TITLE
RuntimeError: Error(s) in loading state_dict for CoTrackerThreeOffline:

### ISSUE THREAD

**Issue by Au-LiuJY** [MC-E/ReVideo#8]
I download the cotracker2.pth locally and move it into the /.cache/torch/hub/checkpoints/ to load.But when I run the demo "bash configs/examples/constant_motion/head6.sh",some errors occurs.

RuntimeError: Error(s) in loading state_dict for CoTrackerThreeOffline:
        Missing key(s) in state_dict: "updateformer.vis_conf_head.weight", "updateformer.vis_conf_head.bias", "corr_mlp.fc1.weight", "corr_mlp.fc1.bias", "corr_mlp.fc2.weight", "corr_mlp.fc2.bias".
        ……
        size mismatch for time_emb: copying a param with shape torch.Size([1, 8, 456]) from checkpoint, the shape in current model is torch.Size([1, 8, 1110]).
        size mismatch for updateformer.input_transform.weight: copying a param with shape torch.Size([384, 456]) from checkpoint, the shape in current model is torch.Size([384, 1110]).
        size mismatch for updateformer.flow_head.weight: copying a param with shape torch.Size([130, 384]) from checkpoint, the shape in current model is torch.Size([2, 384]).
        size mismatch for updateformer.flow_head.bias: copying a param with shape torch.Size([130]) from checkpoint, the shape in current model is torch.Size([2]).

Can anyone help me solve this problem?Thanks in advance!

**Comment by Weiwenliang123** [comment]
I also encountered this issue.

**Comment by MC-E** [comment]
Did you load cotracker with this command? cotracker = torch.hub.load("facebookresearch/co-tracker", "cotracker2").to(device)

**Comment by Weiwenliang123** [comment]
> Did you load cotracker with this command? cotracker = torch.hub.load("facebookresearch/co-tracker", "cotracker2").to(device)

Yes.

**Comment by MC-E** [comment]
Yes, this issue does exist. I reproduce it. This is a version issue https://github.com/facebookresearch/co-tracker/issues/123#issuecomment-2443552717.

**Comment by tissueboxxes** [comment]
I also encountered this issue, but I load cotracker with this command.
cotracker = CoTrackerPredictor(checkpoint=args.checkpoint).to(args.device)

**Comment by tissueboxxes** [comment]
I also encountered this issue, but I load cotracker with this command:
cotracker = CoTrackerPredictor(checkpoint=args.checkpoint).to(args.device)
I changed the checkpoint from  cotracker_stride_4_wind_8.pth to scaled_offline.pth, and the problem was sloved.
